addons/Page_Progress_Panel/src/presenter.js

```diff
@@ -111,9 +109,7 @@ function AddonPage_Progress_Panel_create(){
 		presenter.displayScores(presenter.lastScores);
 	};
 
-    presenter.destroy = function (event) {
-        presenter.view.removeEventListener('DOMNodeRemoved', presenter.destroy);
-    };
+    presenter.destroy = function (event) { };
```

Comment:
Ten destroy nic nie robi. Myślę, że nie ma co go zostawiać, tylko skoro i tak są tutaj zmiany, to można to posprzątać i albo dać destruktor który coś rzeczywiście robi, albo po prostu wywalić.